esmvalcore/cmor/check.py

```diff
@@ -431,8 +433,23 @@ def _check_coord_values(self, coord_info, coord, var_name):
                                       '> {} ='.format('valid_max'), valid_max)
 
         if l_fix_coord_value:
-            lon_extent = iris.coords.CoordExtent(coord, 0.0, 360., True, False)
-            self._cube = self._cube.intersection(lon_extent)
+            if coord.ndim == 1:
+                lon_extent = iris.coords.CoordExtent(
+                    coord, 0.0, 360., True, False)
+                self._cube = self._cube.intersection(lon_extent)
+            else:
+                new_lons = coord.points.copy()
+                new_bounds = coord.bounds.copy()
+                while new_lons.min() < 0:
+                    new_lons[new_lons < 0] += 360
+                    new_bounds[new_bounds < 0] += 360
+                while new_lons.max() > 360:
+                    new_lons[new_lons > 360] -= 360
+                    new_bounds[new_bounds > 360] -= 360
+                new_coord = coord.copy(new_lons, new_bounds)
+                dims = self._cube.coord_dims(coord)
+                self._cube.remove_coord(coord)
```

Comment:
this is not working as expected (see failed tests) - units need to change and you are adding eithet `lat` or `lon` as auxiliary coordinate without adding the other as auxiliary coordinate too

Reply:
actually, forget about it, the 2 failing tests are not well written for this new implementation: you are giving lon points to lat so of course it fails :grin: